MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -255,6 +255,19 @@
                 }
 				break;
 #endif
+			case VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_DRIVER_PROPERTIES: {
+				auto* physicalDeviceDriverProps = (VkPhysicalDeviceDriverPropertiesKHR*)next;
+				char mvkVersion[16], vkVersion[16];
+				vkGetVersionStringsMVK(mvkVersion, sizeof(mvkVersion), vkVersion, sizeof(vkVersion));
+				strcpy(physicalDeviceDriverProps->driverName, "MoltenVK");
+				strcpy(physicalDeviceDriverProps->driverInfo, mvkVersion);
+				physicalDeviceDriverProps->driverID = (VkDriverId) 0; /* FIXME: VK_DRIVER_ID_MOLTENVK_KHR */
+				physicalDeviceDriverProps->conformanceVersion.major = VK_VERSION_MAJOR(VK_API_VERSION_1_0);
+				physicalDeviceDriverProps->conformanceVersion.minor = VK_VERSION_MINOR(VK_API_VERSION_1_0);
+				physicalDeviceDriverProps->conformanceVersion.subminor = 0; /* FIXME: What is this? */
+				physicalDeviceDriverProps->conformanceVersion.patch = VK_VERSION_PATCH(VK_HEADER_VERSION);
```

Comment:
So this is actually supposed to be the version of the [Vulkan conformance test suite](/KhronosGroup/VK-GL-CTS) that the driver has been validated against. To my knowledge, MoltenVK has not yet passed verification against any version of the CTS... so I'm not sure what to actually put here.

Reply:
Hmm. Maybe 0.0.0.0 then?